src/modules/UI/scenes/OnBoarding/OnBoardingScene.js

```diff
@@ -0,0 +1,111 @@
+// @flow
+import React, { Component } from 'react'
+import { Animated, Modal, View } from 'react-native'
+
+import PagingDots from '../../../../connectors/components/OnboardingPagingDotsConnector.js'
+import s from '../../../../locales/strings.js'
+import { OnBoardingSceneStyles } from '../../../../styles/indexStyles.js'
+import { PLATFORM } from '../../../../theme/variables/platform'
+import { PrimaryButton } from '../../components/Buttons/'
```

Comment:
You should be using `PrimaryButton` from `/components/Modals/components/PrimaryButton.ui.js`. We are going to be eventually deprecating the button component that you used.